src/Test/Test.csproj

```diff
@@ -6,6 +6,7 @@
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <AssetTargetFallback>portable-net45+win8</AssetTargetFallback>
     <NoWarn>NU1701;NU1603;NU1603;</NoWarn>
+    <Version>$(Version)</Version>
```

Comment:
do we need this for L0 test project? we don't ship it to customer. :)

Reply:
Not really, I'll revert this change.